eacl/target.go

```diff
@@ -45,7 +47,26 @@ func (t Target) CopyTo(dst *Target) {
 //
 // The value returned shares memory with the structure itself, so changing it can lead to data corruption.
 // Make a copy if you need to change it.
+// Deprecated: use [Target.Accounts] instead.
```

Comment:
sounds a little strange to me: if i am using binary keys, using `Accounts` does not solve my "deprication-message-from-linter" problem; can be like "using raw binary keys is not encouraged and will be dropped eventually, use Accoun-based targets instead". however, it is a minor problem